Public/Src/Engine/Processes/SandboxedProcessPipExecutor.cs

```diff
@@ -673,6 +673,23 @@ public async Task<SandboxedProcessPipExecutionResult> RunAsync(CancellationToken
                     string arguments = m_pip.Arguments.ToString(m_pipDataRenderer);
                     m_timeout = GetEffectiveTimeout(m_pip.Timeout, m_sandboxConfig.DefaultTimeout, m_sandboxConfig.TimeoutMultiplier);
 
+                    // If ChangeAffectedInputListWrittenFilePath is set, write the change affected inputs of the pip to the file.
+                    if (m_pip.ChangeAffectedInputListWrittenFilePath != default)
+                    {
+                        var changeAffectedInputs = m_pip.GetChangeAffectedInputNames(m_pathTable);

```

Comment:
>GetChangeAffectedInputNames [](start = 57, length = 27)

Did I infer correctly that this does not yet have the transitive pattern?

Reply:
It includes the transitive pattern.